LuciBase.py

```diff
@@ -1239,3 +1239,69 @@ def check_luci_path(self):
             self.Luci_path += '/'
             print("We have added a trailing '/' to your Luci_path variable.\n")
             print("Please add this in the future.\n")
+     
+    def fit_wvt(self, x_min, x_max, y_min, y_max, lines, fit_function, vel_rel, sigma_rel, pixel_size, StN_target, roundness_crit, ToL, bkg=None, bayes_bool=False, uncertainty_bool=False, mean=False, n_threads=1):
```

Comment:
We should break this into 2 different functions: the first to create the WVT bins and the second to fit the bins. That way if people only wanted the WVT bins then they would have it. Also, the second function to fit the bins should make maps for all the output parameters not just the velocity.